doc/threat-model.md

```diff
@@ -0,0 +1,9 @@
+Until there is a more concise explanation here about the threat model (what it
```

Comment:
Does this really have to be  a separate file? Both SECURITY.md and threat-model.md are tiny, this feels like over-splitting.